computetemplates/gke-sandbox.yaml

```diff
@@ -10,12 +10,12 @@ spec:
   - diskSize: 30
     autoscaling:
       enabled: true
-      maxSize: 3
+      maxSize: 10
```

Comment:
that seems inexplicably large for a sandbox?

Reply:
it does - lets revert this thing from the codebase 😨 ⚖️ 🎫 🐯